index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+
+    <link rel="stylesheet" href="css/style.css" />
+    <!-- Responsive Stylesheet -->
+    <link rel="stylesheet" href="css/responsive.css" />
+  </head>
+  <body>
+    <header>
+      <div class="header">
+        <div class="nav-bar">
+          <div class="nav-bar-border">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
```

Comment:
I don't fully know your intention, but I think this could be simplified  as well, also maybe a better `class` name would be `hamburger-icon` or something like that